hbase-common/src/main/java/org/apache/hadoop/hbase/trace/HBaseSemanticAttributes.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hbase.trace;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import java.util.List;
+import org.apache.yetus.audience.InterfaceAudience;
+
+/**
+ * The constants in this class correspond with the guidance outlined by the OpenTelemetry
+ * <a href="https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions">Semantic Conventions</a>.
+*/
+@InterfaceAudience.Private
+public final class HBaseSemanticAttributes {
+  public static final AttributeKey<String> NAMESPACE_KEY = SemanticAttributes.DB_HBASE_NAMESPACE;
```

Comment:
[nit] seems like they have deprecated it after [opentelemetry 1.8.0](https://github.com/open-telemetry/opentelemetry-java/blob/main/semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java#L1241-L1248), and they ask user to use a generic variable of `SemanticAttributes#DB_NAME`, for long term, what should it be ?

Reply:
nvm, although I'm confused with their versioning now, I found [their v1.9.1 still have this variable](https://github.com/open-telemetry/opentelemetry-java/blob/v1.9.1/semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java#L168-L172), if you have a chance, can you reach out and ask them before pushing ?

Reply:
It seems their intent is to drop these system-specific attributes in favor of the generic attribute. Sure, I'll ask.

Reply:
https://cloud-native.slack.com/archives/C0150QF88FL/p1638287586272800

Reply:
FYI, the next patch will introduce use of the attribute `db.name`: https://github.com/apache/hbase/pull/3863/commits/84ea8e2eecc417d59610cef7a29f1c98c84ec851#diff-aac65169b0283bbffad891c962709e203e22d6a5170d160ab72ee9e9f6cbbe3cR31